tooling/scripts/validate-github-workflows.groovy

```diff
@@ -30,7 +30,7 @@ final Set<String> executedBaseNames = [] as Set
 
 // Add any ignored itest modules here. Or prefix the module name with '#' to disable it
 // hystrix - https://github.com/apache/camel-quarkus/issues/1146
-final List<String> excludedModules = ['hystrix', 'nats'] as List
+final List<String> excludedModules = ['hystrix', 'messaging', 'nats'] as List
```

Comment:
Wouldn't integration-tests-support be a better place for the messaging module, given that it is not a real integration test? We could  avoid excluding it here and in tooling/test-list/pom.xml then. But you maybe have some reason to keep `messaging` under `integration-tests`?

Reply:
BTW, do we still need to exclude nats here? I mean it is there in test-categories.yaml

Reply:
> But you maybe have some reason to keep `messaging` under `integration-tests`?

Mainly to avoid having to navigate away from the main itests area if you want to add / amend tests. If it were just  few tests that we were maintainting, I'd probably have put it under test-support. Since its a catalog of tests that we'll likely add to in future, I thought i made sense to keep them in the standard location.